api/src/pcapi/alembic/versions/20211115T073416_a0a822d365ac_add_not_null_venue_provider_venue_id_at_offer_provider_step_1.py

```diff
@@ -0,0 +1,23 @@
+"""add_not_null_venue_provider_venue_id_at_offer_provider
```

Comment:
(micro-détail)

```suggestion
"""Add NOT NULL on venue_provider.venueIdAtOfferProvider (step 1)
```

... et idem pour la docstring des autres migrations. C'est ce que qu'utilise `alembic history` pour afficher une "description" de la migration, ça vaut donc le coup de le personnaliser un petit peu. :)